app/helpers/user_helper.rb

```diff
@@ -0,0 +1,11 @@
+module UserHelper
+  def added_friend(user, friendship)
+    if !current_user.id == user.id && !Friendship.where(user_id: current_user.id,
+                                                        friend_id: user.id).or(Friendship.where(
+                                                                                 friend_id: current_user.id,
+                                                                                 user_id: user.id
+                                                                               )).exists?
```

Comment:
you should never use the `where` method, or any other query-building methods like this, outside of the model class itself, Take a look at this article for more Ruby on Rails best practices: [Buggy Rails Code: The 10 Most Common Mistakes That Rails Developers Make](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make)